src/bp/nlu-core/predict-pipeline.ts

```diff
@@ -122,22 +115,26 @@ const getCustomEntitiesNames = (predictors: Predictors): string[] => [
   ...predictors.pattern_entities.map(e => e.name)
 ]
 
-async function predictContext(input: OutOfScopeStep, predictors: Predictors): Promise<ContextStep> {
-  const customEntities = getCustomEntitiesNames(predictors)
-
+export async function predictContext(input: OutOfScopeStep, predictors: Predictors): Promise<ContextStep> {
   const classifier = predictors.ctx_classifier
   if (!classifier) {
+    const ctxCount = predictors.contexts.length
```

Comment:
This condition only occurs when there's 0 or 1 context with utterances.

In this case we don't want to elect anything, we let the election logic do its job.